atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/atomic/ResilientCommitTimestampAtomicTableTest.java

```diff
@@ -226,7 +264,7 @@ public void noSuperfluousCasOrPuts() {
         spiedStore.stopFailingPuts();
         ExecutorService writers = Executors.newFixedThreadPool(100);
         ThreadLocalRandom random = ThreadLocalRandom.current();
-        List<Future<ListenableFuture<Long>>> results = new ArrayList<>();
+        List<Future<ListenableFuture<?>>> results = new ArrayList<>();
```

Comment:
`TransactionStatus`? We should still be strict with the type check.

Reply:
yep! how you caught this is incredible!